README.md

```diff
@@ -3,15 +3,19 @@
 A django app that replaces the django admin authentication mechanism by
 deferring to an oauth2 provider.
 
-[![Build Status](https://travis-ci.org/RealGeeks/django-admin-oauth2.png?branch=master)](https://travis-ci.org/RealGeeks/django-admin-oauth2)
+[![Build Status](https://travis-ci.org/bastionhost/django-admin-oauth2.png?branch=master)](https://travis-ci.org/bastionhost/django-admin-oauth2)
 
 # Support
 
 django-admin-oauth2 should support Python 2.7, pypy, and Django versions 1.6 through 1.10
 
 ## Installation
 
-Step 1: `pip install django-admin-oauth2` and include it in your project's requirements
+Integrating Demo: https://github.com/huifenqi/dbhub/pull/1/files
+
+![workflow](./workflow.png)
+
+Step 1: `pip install git+ssh://git@github.com/bastionhost/django-admin-oauth2.git` and include it in your project's requirements
```

Comment:
Why is this pointing to a fork?
Better provide testing repo's URL on PR description, if needed.